### PR TITLE
Badge form

### PR DESCRIPTION
**Pull request by mikolajmirko**
(based on award page - so first we need to close that - if there will be any changes Ill pull them here)

### CODE REVIEW

File: i18n/en_US.json [hunk 1]
```diff
@@ -187,7 +187,7 @@
 				"saveAwardButton": "Save award",
 				"removeAwardButton": "Remove award",
 				"removeAwardTitle": "Remove award",
-				"removeAwardText": "Are you sure you want to remove this wawrd?",
```
Comment (stasgora) [inline i18n/en_US.json:190]: :smile:

File: i18n/en_US.json [hunk 2]
```diff
@@ -355,6 +389,7 @@
 		"plan_creation": "Plan creation",
 		"task_creation": "Task creation",
 		"award_creation": "Award creation",
+		"badge_creation": "Tworzenie odznaki",
```
Comment (stasgora) [inline i18n/en_US.json:392]: Language

File: lib/model/ui/award/ui_badge.dart [hunk 3]
```diff
@@ -0,0 +1,17 @@
+
+enum UIBadgeLevel { one, three, five }
+
+class UIBadge {
+	String name;
+	String description;
+	UIBadgeLevel level;
```
Comment (stasgora) [inline lib/model/ui/award/ui_badge.dart:7]: maxLevel?

File: lib/model/ui/award/ui_badge.dart [hunk 4]
```diff
@@ -0,0 +1,17 @@
+
+enum UIBadgeLevel { one, three, five }
```
Comment (stasgora) [inline lib/model/ui/award/ui_badge.dart:2]: We will probably need int values of those so add an extension

**Review by stasgora — CHANGES_REQUESTED** [review]
:+1:

File: i18n/en_US.json (hunk 1 above)
Comment (mikolajmirko) [reply i18n/en_US.json:190]: Oh, I thought I somehow did it wrong again! :P

**Review by stasgora — APPROVED** [review]
Just resolve the conflicts

**Comment by mikolajmirko** [comment]
Pulled from master and fixed conflicts. Merging pull request.

**Pull request merged**